### PR TITLE
Update README.md Co-authored-by: Sang Vo <stvo1589@gmail.com>

### PR DESCRIPTION
**Pull request by yauacom**
Update README.md Co-authored-by: Sang Vo <stvo1589@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ This project is a website designed to support students on their journey through
 9. [Kết Luận](#headingNine)
 
 - **Additional Resources**:
-  - [Cách Đặt Câu Hỏi](./askingTechnicalQuestions.html)
+  - [Cách Đặt Câu Hỏi](https://sangvo117.github.io/Motivation-and-Mindset/askingTechnicalQuestions.html)
```
Comment (sangvo117) [inline README.md:28]: Why do we need this modification?

**Comment by sangvo117** [comment]
Why do we need this modificaiton?

**Comment by yauacom** [comment]
User can navigate to that specific web page.

**Comment by sangvo117** [comment]
Look good

File: README.md (hunk 1 above)
Comment (sangvo117) [reply README.md:28]: Looks good

**Pull request merged**